rviz_simulator/config/david_camera.yaml

```diff
@@ -0,0 +1,20 @@
+name: "David's Camera (Camera)"
+
```

Comment:
Since we switched to using exclusively [`sensor_msgs/CameraInfo Message`](http://docs.ros.org/melodic/api/sensor_msgs/html/index-msg.html), we no longer use this YAML file structure, will remove it from the repo.